### DISCUSSION TITLE
Ideas for new features ... !✨add on yours below.

### DISCUSSION THREAD

**Discussion by VAbhijith2003github** [VAbhijith2003github/SIH#1] [Ideas]
<h1>IDEAS FOR THE FEATURES NEEDED</h1>
 @Jis369 
 @myselfchandu  
Add on your ideas for new features in this discussion section
👍🧩✨
ex:
In user login area

- file upload for certificate
- file upload for user profile photo
- verification status indicator
..... et cetera

**Comment by Jis369** [comment]
Flow of work

1.Login page 
   i)  login with email and password
2. Registraion profile 

- profile pic
- Name
- dob
- gender
- category
- Citizen
- Registered email
- alternate email
- mobile
- alternate mobile
- permanent address
- correspondence address
- id proof (options)
- id proof number
-  place of birth details - city/district/state/country
- mothers name
- fathers name
- emergency contact number
- educational qualification (highest degree obtained)
- year of passing
- school/college name
- name of board
- upload certificate
- Disability details
- proof of disability 
- income certificate(optional)

3.wait for the official confirmation from govt side via message /mail/call

@VAbhijith2003github @myselfchandu

**Reply by VAbhijith2003github** [reply]
These parameters will be the fields for the user register page input and additionally will be needing aadhaar and disability certificate upload area to cloud bucket 
@Jis369 @myselfchandu
